code/game/machinery/requests_console.dm

```diff
@@ -60,6 +60,9 @@ GLOBAL_LIST_EMPTY(allConsoles)
 	obj_integrity = 300
 	max_integrity = 300
 	armor = list(melee = 70, bullet = 30, laser = 30, energy = 30, bomb = 0, bio = 0, rad = 0, fire = 90, acid = 90)
+	var/distress_state = 0

```

Comment:
This needs to be a global var. Unless I'm mistaken, I don't see anything that's preventing someone from simply triggering the distress on every console simultaneously.